Libraries/xcsdk/Headers/xcsdk/SDK/Manager.h

```diff
@@ -89,6 +89,12 @@ class Manager {
      */
     std::vector<std::string> executablePaths() const;
 
+    /*
+     * Conglomeration of executable paths that optionally includes extra toolchains
+     * and the paths from an SDK target.
+     */
+    std::vector<std::string> executablePaths(std::vector<xcsdk::SDK::Toolchain::shared_ptr> &toolchains, xcsdk::SDK::Target::shared_ptr &target) const;
```

Comment:
I can certainly get rid of this -- the main thing I wanted to make sure was added was all of the other stuff that target.executablePaths() adds, like the platform's paths and the manager's paths.